<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="41872" started-at="2015-11-29T04:40:26Z" finished-at="2015-11-29T04:41:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="41872" started-at="2015-11-29T04:40:26Z" finished-at="2015-11-29T04:41:07Z">
      <class name="com.test.testcases.Login">
        <test-method status="PASS" signature="initialDriver()[pri:0, instance:com.test.testcases.Login@763d9750]" name="initialDriver" is-config="true" duration-ms="8862" started-at="2015-11-29T12:40:26Z" finished-at="2015-11-29T12:40:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialDriver -->
        <test-method status="FAIL" signature="testLogin(java.util.Map)[pri:0, instance:com.test.testcases.Login@763d9750]" name="testLogin" duration-ms="32965" started-at="2015-11-29T12:40:34Z" data-provider="providerMethod" finished-at="2015-11-29T12:41:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=123456, url=https://www.jd.com, username=test1}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.test.testcases.Login.testLogin(Login.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
      </class> <!-- com.test.testcases.Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
